{
  "uid" : "ab02803e851e65c",
  "name" : "User can't successfully log in with invalid username.",
  "fullName" : "tests.core_features.test_login.TestLogin#test_login_with_invalid_username",
  "historyId" : "d9da1d35f9d1a309fb04ddfe89a222bc",
  "time" : {
    "start" : 1705147685894,
    "stop" : 1705147703363,
    "duration" : 17469
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.TimeoutException: Message:",
  "statusTrace" : "tests/core_features/test_login.py:69: in test_login_with_invalid_username\n    login_page.error_text.assert_error(\"The username and password could not be verified.\")\n        driver     = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0b62a63aeea5ba637f244ad3645a2b3a\")>\n        home_page  = <src.pages.home_page.HomePage object at 0x7fdbd54832d0>\n        login_page = <src.pages.login_page.LoginPage object at 0x7fdbd5481910>\n        self       = <test_login.TestLogin object at 0x7fdbd6c42a90>\nsrc/base/base_element.py:69: in assert_error\n    current_error = self.get_text()\n        expected_error = 'The username and password could not be verified.'\n        self       = <src.base.base_element.BaseElement object at 0x7fdbd54dc850>\n        server_error = 'An internal error has occurred and has been logged.'\nsrc/base/base_element.py:37: in get_text\n    element = self.assert_element()\n        self       = <src.base.base_element.BaseElement object at 0x7fdbd54dc850>\nsrc/base/base_element.py:16: in assert_element\n    self.wait.until(EC.presence_of_element_located((\"xpath\", self.xpath)))\n        clickable  = False\n        return_many = False\n        self       = <src.base.base_element.BaseElement object at 0x7fdbd54dc850>\n../local/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: in until\n    raise TimeoutException(message, screen, stacktrace)\nE   selenium.common.exceptions.TimeoutException: Message:\n        end_time   = 357.584785966\n        message    = ''\n        method     = <function presence_of_element_located.<locals>._predicate at 0x7fdbd6c66fc0>\n        screen     = None\n        self       = <selenium.webdriver.support.wait.WebDriverWait (session=\"0b62a63aeea5ba637f244ad3645a2b3a\")>\n        stacktrace = None",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "Health check request",
    "time" : {
      "start" : 1705147517847,
      "stop" : 1705147518594,
      "duration" : 747
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1705147517826,
      "stop" : 1705147517847,
      "duration" : 21
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "Initialize driver",
    "time" : {
      "start" : 1705147685055,
      "stop" : 1705147685893,
      "duration" : 838
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message:",
    "statusTrace" : "tests/core_features/test_login.py:69: in test_login_with_invalid_username\n    login_page.error_text.assert_error(\"The username and password could not be verified.\")\n        driver     = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0b62a63aeea5ba637f244ad3645a2b3a\")>\n        home_page  = <src.pages.home_page.HomePage object at 0x7fdbd54832d0>\n        login_page = <src.pages.login_page.LoginPage object at 0x7fdbd5481910>\n        self       = <test_login.TestLogin object at 0x7fdbd6c42a90>\nsrc/base/base_element.py:69: in assert_error\n    current_error = self.get_text()\n        expected_error = 'The username and password could not be verified.'\n        self       = <src.base.base_element.BaseElement object at 0x7fdbd54dc850>\n        server_error = 'An internal error has occurred and has been logged.'\nsrc/base/base_element.py:37: in get_text\n    element = self.assert_element()\n        self       = <src.base.base_element.BaseElement object at 0x7fdbd54dc850>\nsrc/base/base_element.py:16: in assert_element\n    self.wait.until(EC.presence_of_element_located((\"xpath\", self.xpath)))\n        clickable  = False\n        return_many = False\n        self       = <src.base.base_element.BaseElement object at 0x7fdbd54dc850>\n../local/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: in until\n    raise TimeoutException(message, screen, stacktrace)\nE   selenium.common.exceptions.TimeoutException: Message:\n        end_time   = 357.584785966\n        message    = ''\n        method     = <function presence_of_element_located.<locals>._predicate at 0x7fdbd6c66fc0>\n        screen     = None\n        self       = <selenium.webdriver.support.wait.WebDriverWait (session=\"0b62a63aeea5ba637f244ad3645a2b3a\")>\n        stacktrace = None",
    "steps" : [ {
      "name" : "Navigate to Home page",
      "time" : {
        "start" : 1705147685894,
        "stop" : 1705147686881,
        "duration" : 987
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Fill in valid password, but invalid username",
      "time" : {
        "start" : 1705147686882,
        "stop" : 1705147687041,
        "duration" : 159
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Log in",
      "time" : {
        "start" : 1705147687041,
        "stop" : 1705147688158,
        "duration" : 1117
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Check error message is displayed",
      "time" : {
        "start" : 1705147688158,
        "stop" : 1705147703362,
        "duration" : 15204
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \n\n",
      "statusTrace" : "  File \"/usr/workspace/tests/core_features/test_login.py\", line 69, in test_login_with_invalid_username\n    login_page.error_text.assert_error(\"The username and password could not be verified.\")\n  File \"/usr/workspace/src/base/base_element.py\", line 69, in assert_error\n    current_error = self.get_text()\n                    ^^^^^^^^^^^^^^^\n  File \"/usr/workspace/src/base/base_element.py\", line 37, in get_text\n    element = self.assert_element()\n              ^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/workspace/src/base/base_element.py\", line 16, in assert_element\n    self.wait.until(EC.presence_of_element_located((\"xpath\", self.xpath)))\n  File \"/usr/local/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 101, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "Initialize driver::0",
    "time" : {
      "start" : 1705147703369,
      "stop" : 1705147703426,
      "duration" : 57
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Terminate driver",
      "time" : {
        "start" : 1705147703369,
        "stop" : 1705147703426,
        "duration" : 57
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "PARA-15240: User registration and authentication"
  }, {
    "name" : "feature",
    "value" : "PARA-15261: User login"
  }, {
    "name" : "severity",
    "value" : "Minor"
  }, {
    "name" : "parentSuite",
    "value" : "tests.core_features"
  }, {
    "name" : "suite",
    "value" : "test_login"
  }, {
    "name" : "subSuite",
    "value" : "TestLogin"
  }, {
    "name" : "host",
    "value" : "fdd3900bcbbf"
  }, {
    "name" : "thread",
    "value" : "1-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.core_features.test_login"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "ab02803e851e65c.json",
  "parameterValues" : [ ]
}